scripts/update-L10N-files

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+#
+# Run over all translated .properties files for a single language,
+# updating using PropertiesMergeTool
+#
+# Takes one argument, which is the language to be updated e.g. "es" or "en_GB"
+#
+# Only updates if the translation file already exists
+#
+
+for filename in `find java/src -name \*_${1}.properties`
+do
+    basefile=${filename%%_es.*}.properties
```

Comment:
This line doesn't look right.
basefile=${filename%%**__es_**.*}.properties

I'm approving this PR now and if you want to include it in the next test release 4.99.9, feel free to do so. This issue can be fixed in another PR.